Source/Views/TableViews/SPCopyTable.m

```diff
@@ -1234,13 +1234,25 @@ - (BOOL)shouldUseFieldEditorForRow:(NSUInteger)rowIndex column:(NSUInteger)colIn
 {
 	// Return YES if the multiple line editing button is enabled - triggers sheet editing on all cells.
 	if ([prefs boolForKey:SPEditInSheetEnabled]) return YES;
-	
+
+    NSMutableDictionary *preferenceDefaults = [NSMutableDictionary dictionaryWithContentsOfFile:[[NSBundle mainBundle] pathForResource:SPPreferenceDefaultsFile ofType:@"plist"]];
+
+    NSUInteger editInSheetForLongTextLengthThreshold = (NSUInteger)[preferenceDefaults safeObjectForKey:@"ssfsfs"];
+
+    if([prefs boolForKey:SPEditInSheetForLongText] && [prefs objectForKey:SPEditInSheetForLongTextLengthThreshold]){
```

Comment:
This still doesn't check if `[prefs objectForKey:SPEditInSheetForLongTextLengthThreshold]` is `integerValue` as you are getting on next line... should we do the check?